vispTypes.hpp

```diff
@@ -38,6 +39,23 @@ namespace visp
         float gain_at_infinity;
         float slope_at_zero;
     };
+
+    /** Group relevant information about the controller
+     *
+     *  error        - the norm-2 of the error vector
+     *  residual     - value expressed in meter that represents the
+                       "quality" of the measurement"
+     *  desired_pose - the current setpoint
+     *  current_pose - the current pose
+     */
+    struct controllerState
+    {
+        double error;
```

Comment:
> Do you think that is important to inform it explicitly?

It is, because the logging is then using this time as the "sample log time". I do agree that it is suboptimal since

However, one virtually never uses the time-of-writing as the timestamp. You need to find a good logical time (which in this case would probably be the timestamp of the last received visual sample).